modules/github_reverse_proxy/sg.tf

```diff
@@ -0,0 +1,25 @@
+resource "aws_security_group" "lambda_sg" {
```

Comment:
No ingress needed, becuase lambda is serverless. Event comes from AWS internal service-to-service network.